### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by micrococo**

### CODE REVIEW

File: locales/es.json [hunk 1]
```diff
@@ -0,0 +1,294 @@
+{
+  "`x` subscribers": "`x` suscriptores",
+  "`x` videos": "`x` vídeos",
+  "LIVE": "DIRECTO",
+  "Shared `x` ago": "Compartido hace `x`",
+  "Subscribe": "Desuscribirse",
```
Comment (omarroth) [inline locales/es.json]: Appears to be missing `Unsubscribe`. Should be something like:
```json
  "Unsubscribe": "Desuscribirse",
  "Subscribe": "Suscribirse",
```

**Comment by omarroth** [comment]
A minor fix and this should be ready to merge :+1:.

**Comment by micrococo** [comment]
@omarroth, fixed.

**Comment by omarroth** [comment]
LGMT, thanks!

**Pull request merged**